testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusConfiguration.java

```diff
@@ -22,7 +22,7 @@ public class KeycloakQuarkusConfiguration implements ContainerConfiguration {
     private int bindHttpPortOffset = 100;
     private int bindHttpPort = 8080;
     private int bindHttpsPortOffset = 0;
-    private int bindHttpsPort = Integer.valueOf(System.getProperty("auth.server.https.port", "8543"));
+    private int bindHttpsPort = Integer.parseInt(System.getProperty("auth.server.https.port", "8543"));
```

Comment:
A nitpick and optional suggestion.

As you are changing here, perhaps we can just use `java.lang.Integer#getInteger(java.lang.String, java.lang.Integer)`.

Reply:
Fixed, thanks.